README.md

```diff
@@ -387,6 +391,10 @@ buildctl build ... \
   --import-cache type=gha
 ```
 
+Github Actions cache saves both cache metadata and layers to GitHub's Cache service. This cache currently has a [size limit of 10GB](https://docs.github.com/en/actions/advanced-guides/caching-dependencies-to-speed-up-workflows#usage-limits-and-eviction-policy) that is shared accross different caches in the repo. If you exceed this limit, GitHub will save your cache but will begin evicting caches until the total size is less than 10 GB. Recycling caches too often can result in slower runtimes overall.
+
+Similarly to using [actions/cache](https://github.com/actions/cache), caches are scoped by branch, with the default branch being available to every branch. E.g. a pull request to the default branch can load cache generated by the default branch but it can not overwrite that cache. Caches generated in pull requests are only used when the same pull request needs to be rebuilt. Caches generated for the default branch are used in next default branch builds and for all pull requests.
```

Comment:
> pull request to the default branch can load cache generated by the default branch

Is there a doc link for this? I thought this was the target branch of the PR, not the default branch. But I have never verified it.

Reply:
It seems to be both actually https://docs.github.com/en/actions/advanced-guides/caching-dependencies-to-speed-up-workflows#restrictions-for-accessing-a-cache:

> A workflow can access and restore a cache created in the current branch, the base branch (including base branches of forked repositories), or the default branch (usually `main`). For example, a cache created on the default branch would be accessible from any pull request. Also, if the branch `feature-b` has the base branch `feature-a`, a workflow triggered on `feature-b` would have access to caches created in the default branch (`main`), `feature-a`, and `feature-b`.

Scope priority is not specified but I would guess `feature-b` > `feature-a` > `main`.

Reply:
I replaced the examples with this reference.